### ISSUE TITLE
void

### ISSUE THREAD

**Issue by lazaridiscom** [bitcoin/bitcoin#15184]

**Comment by fanquake** [comment]
@lazaridiscom I'm pretty sure there are tags available? Can you do:

`git checkout v0.1.5`:
```
Note: checking out 'v0.1.5'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 8dca7864f No dll's here
```

`git log`:
```
commit 8dca7864f793072701f810e4c5ea12a6e1087085 (HEAD, tag: v0.1.5)
Author: sirius-m <sirius-m@1a98c847-1fd6-4fd8-948a-caf3550aa51b>
Date:   Wed Sep 16 13:26:04 2009 +0000

    No dll's here
    
    git-svn-id: https://bitcoin.svn.sourceforge.net/svnroot/bitcoin/trunk@8 1a98c847-1fd6-4fd8-948a-caf3550aa51b

commit 465e1d99f25cc7f45efcacfcbb1f95c1c1f9a4ee
Author: sirius-m <sirius-m@1a98c847-1fd6-4fd8-948a-caf3550aa51b>
Date:   Sun Aug 30 03:50:53 2009 +0000

    Added changelog.txt
    
    
    git-svn-id: https://bitcoin.svn.sourceforge.net/svnroot/bitcoin/trunk@2 1a98c847-1fd6-4fd8-948a-caf3550aa51b
```

**Comment by lazaridiscom** [comment]
You're right, I checked out the v0.3.0 version an hour ago.

The thing is that I looked into the "Branches/Tags" dropdown (github UI, Branch: ... ), and there it was not listed, so I assumed it does not exist. Then I looked later (after finding a link to this repo again on the net) into the https://github.com/bitcoin/bitcoin/releases and found the older releases.

So all fine, but for future "history-research", possibly this one:

> Additionally, I would be interested to see the "last version where S.N. was the Lead-Dev" is tagged, too (or at least identified somehow).

I could not find this information (the last release from Satoshi N.).

Possibly some of the devs knows?

(As for documenting this here in this repo: a tiny section in the README could do it, or even a separate file "SATOSHI.md" in root. Not of any priority of course.)
